mopidy_alsamixer/mixer.py

```diff
@@ -143,8 +186,7 @@ def volume_to_mixer_volume(self, volume):
     def get_mute(self):
         try:
             channels_muted = self._mixer.getmute()
-        except alsaaudio.ALSAAudioError as exc:
-            logger.debug(f"Getting mute state failed: {exc}")
```

Comment:
I removed it for consistency (`get_volume` has no such logging) and because I thought this method (as well as `get_volume`) may be called repeatedly generating lots of error messages due to the actual error being suppressed.

I'm grateful you bring this up because now I think it's better to do the opposite and add logging to `get_volume` too, as `ALSAAudioError` can occur during `alsaaudio.Mixer.getvolume`, not only during `alsaaudio.Mixer`.